packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -46,6 +43,20 @@ if ([production, development].indexOf(mode) < 0) {
     throw new Error(`Invalid mode "${mode}"`)
 }
 
+if (pkg?.ccExtensibility?.extendable && pkg?.ccExtensibility?.extends) {
```

Comment:
nit, `pkg?.mobify?.overridesDir` is repeated 20 times in this file, have you considered making a variable for it for better readability?